clang/test/SemaSYCL/non-std-layout-param.cpp

```diff
@@ -23,38 +23,11 @@ void test() {
   C C0;
   C0.Y=0;
   kernel_single_task<class MyKernel>([=] {
-    // expected-error@+1 {{kernel parameter has non-standard layout class/struct type}}
+    // expected-error@+1 {{kernel parameter has non-standard layout class/struct type 'C'}}
```

Comment:
Sorry for missing it in my previous comments, but I think it might be useful to check that this test successfully compiles with `-fsycl-new-kernel-param-requirements`.
What do you think?